docs/powerpoint/insert-slides-into-presentation.md

```diff
@@ -0,0 +1,183 @@
+
+    > [!NOTE]
+    > There are many other ways to get a PowerPoint file. For example, if the file is stored on OneDrive or SharePoint, you can use Microsoft Graph to download it. For more information, see [Working with files in Microsoft Graph](/graph/api/resources/onedrive) and [Access Files with Microsoft Graph](/learn/modules/msgraph-access-file-data/).
+
+2. Add the following code to the add-in's JavaScript to assign a function to the input control's `change` event. (You create the `storeFileAsBase64` function in the next step.)
+
+    ```javascript
+    $("#file").change(storeFileAsBase64);
+    ```
+
+3. Add the following code. About this code, note the following:
+
+    - The `reader.readAsDataURL` method converts the file to base64 and stores it in the `reader.result` property. When the method completes, it triggers the `onload` event handler.
+    - The `onload` event handler trims metadata off of the encoded file and stores the encoded string in a global variable.
+    - The base64-encoded string is stored globally because it will be read by another function that you create in a later step.
+
+    ```javascript
+    let chosenFileBase64;
+
+    async function storeFileAsBase64() {
+        const reader = new FileReader();
+
+        reader.onload = async (event) => {
+            const startIndex = reader.result.toString().indexOf("base64,");
+            const copyBase64 = reader.result.toString().substr(startIndex + 7);
+
+            chosenFileBase64 = copyBase64;
+        };
+
+        const myFile = document.getElementById("file") as HTMLInputElement;
+        reader.readAsDataURL(myFile.files[0]);
+    }
+    ```
+
+## Insert slides with insertSlidesFromBase64
+
+You insert slides from another PowerPoint presentation into the current presentation with the [Presentation.insertSlidesFromBase64](/javascript/api/powerpoint/powerpoint.presentation#insertslidesfrombase64-base64file--options-) method. The following is a simple example in which all of the slides from the source presentation are inserted at the beginning of the current presentation and the inserted slides keep the formatting of the source file. Note that `chosenFileBase64` is a global variable that holds a base64-encoded version of a PowerPoint presentation file.
+
+```javascript
+async function insertAllSlides() {
+  await PowerPoint.run(async function(context) {
+    context.presentation.insertSlidesFromBase64(chosenFileBase64);
+    await context.sync();
+  });
+}
+```
+
+You can control some aspects of the insertion result, including where the slides are inserted and whether they get the source or target formatting (and more), by passing an [InsertSlideOptions](/javascript/api/powerpoint/powerpoint.insertslideoptions) object as a second parameter to `insertSlidesFromBase64`. The following is an example. About this code, note:
```

Comment:
Nit: I think the "and more" is unnecessary because saying "including" implies a subset.
```suggestion
You can control some aspects of the insertion result, including where the slides are inserted and whether they get the source or target formatting , by passing an [InsertSlideOptions](/javascript/api/powerpoint/powerpoint.insertslideoptions) object as a second parameter to `insertSlidesFromBase64`. The following is an example. About this code, note:
```